environment.yml

```diff
@@ -11,11 +11,13 @@ dependencies:
   - liquidprompt
   - python 3.7
   - pre-commit
+  - requests
```

Comment:
We use requests to reach out to external registry and download module information during import